extensions/helix/package.build.ts

```diff
@@ -274,6 +274,230 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
+              register: "",
+            }],
+          },
+          "Y": {
+            text: "Yank main selection to clipboard",
+            command: "dance.run",
+            args: [{
+              code: [
+                "const editor = vscode.window.activeTextEditor;",
+                "if (!editor) {",
+                "  return;",
+                "}",
+                "const text = editor.document.getText(editor.selection);",
+                "await vscode.env.clipboard.writeText(text);",
+              ],
+            }],
+          },
+          "p": {
+            text: "Paste clipboard after selections",
+            command: "dance.edit.insert",
+            args: [{
+              handleNewLine: true,
+              where: "end",
+            }],
+          },
+          "P": {
+            text: "Paste clipboard before selections",
+            command: "dance.edit.insert",
+            args: [{
+              handleNewLine: true,
+              where: "start",
+            }],
+          },
+          "R": {
+            text: "Replace selections by clipborad content",
+            command: "editor.action.clipboardPasteAction",
+            args: [],
+          },
+          "/": {
+            text: "Global Search in workspace folder",
+            command: "workbench.action.findInFiles",
+          },
+          "k": {
+            text: "Show docs for item under cursor (hover)",
+            command: "editor.action.showHover",
+          },
+          "r": {
+            text: "Rename symbol",
+            command: "editor.action.rename",
+          },
+          "h": {
+            text: "Select symbol reference",
+            command: "editor.action.referenceSearch.trigger",
+          },
+        },
+
+        "window": {
+          title: "Window",
+          items: {
+            "w": {
+              text: "Goto next window",
+              command: "workbench.action.nextEditor",
+            },
+            "s": {
+              text: "Horizontal bottom split",
+              command: "workbench.action.splitEditorDown",
+            },
+            "v": {
+              text: "Vertical right split",
+              command: "workbench.action.splitEditor",
+            },
+            "t": {
+              text: "Transpose splits",
+              command: "workbench.action.toggleEditorGroupLayout",
+            },
+            // "f": {
+            //   text: "Open files in selection (hsplit)",
+            //   command: "dance.selections.open", // function needs to be modified
+            // },
+            // "F": {
+            //   text: "Open files in selection (vsplit)",
+            //   command: "dance.selections.open", // function needs to be modified
+            // },
+            "q": {
+              text: "Close window",
+              command: "workbench.action.closeActiveEditor",
+            },
+            "o": {
+              text: "Close all other windows (Current window only)",
```

Comment:
nit: the original naming is easier to read IMO

```suggestion
              text: "Close windows except current",
```